### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by rs028**
This branch includes a number of minor updates, fixes, tweaks, and corrections made while working on the new Test Suite:
- updates to the Manual and to the `*.md` files.
- updates to the `.gitignore` files.
- move the mechanism `mcm/mechanism_test.fac` to the `model/` directory and rename it to `mechanism.fac`. This seems a more logical location for an example mechanism.
- rename the `version.sh` script to `update_version_number.sh`.
- move and rename the script to generate the manual pdf file.
- remove extension from license file, as per convention.

### CODE REVIEW

**Comment by rs028** [comment]
@spco Any comments? The idea is to work towards more consistency of the various tools, as well as naming and formatting. I am going to merge this and leave further updates to the manual and readme file for another time.

PS: can the `trigger_action_on_PR_merge` branch be deleted?

**Comment by spco** [comment]
> @spco Any comments? The idea is to work towards more consistency of the various tools, as well as naming and formatting. I am going to merge this and leave further updates to the manual and readme file for another time.
> 
> PS: can the `trigger_action_on_PR_merge` branch be deleted?

I've now deleted this too.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,29 +30,34 @@ jobs:
...
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 2
 
       # Install gfortran 8 since both Ubuntu and macOS don't supply it
       # macOS only
-      - name: Install gfortran-8 on macOS
-        if: matrix.os == 'macos-10.15' && matrix.fortran == 8
-        run: brew install gcc@8
+      #- name: Install gfortran-8 on macOS
```
[the remark this answers is not in the document]
Comment (rs028) [reply .github/workflows/ci.yml:53]: Leftovers from previous version. See the notes I left on the Slack channel about deciding on some guidelines on managing the versions of compiler and OS as they keep becoming obsolete.

**Comment by rs028** [comment]
> Looks like `macos` is now up and running again - I've made it a bit more robust in its handling of `matrix.os` so we don't have to remember to change it in multiple places.
> 
> You're right that `macos-12` only supplies `gfortran-11` out of the box. My concern is that our GH Actions may swiftly outpace the likely target machine for users - and HPC is far more likely to have gfortran-8 than 11! Perhaps we could consider reinstating the `brew install` workaround for macos-12 to install gfortran-9 and gfortran-10? TBH I think `brew install`ing `gfortran-8` on both would also be quite desirable.

this is my concern, and also to keep backwards compatibility but within reason, otherwise it becomes a nightmare to maintain.

File: doc/latex/Development.tex [hunk 2]
```diff
@@ -45,20 +45,20 @@ \section{General Information} \label{sec:general-information}
...
-AtChem2 uses \href{https://github.com/features/actions/}{GitHub Actions} for
+AtChem2 uses \href{https://github.com/features/actions}{GitHub Actions} for
 \textbf{continuous integration} testing. This programming approach
 ensures that changes to the code do not modify the behaviour and the
 results of the software in an unintended fashion. The Test Suite is in
-the \texttt{test/} directory and consists of a series of tests and
+the \texttt{tests/} directory and consists of a series of tests and
 short model runs that check the model functionality and calculations
-against known outputs. The \href{https://codecov.io/}{Codecov} service
+against known outputs. The \href{https://codecov.io}{Codecov} service
```
[the remark this answers is not in the document]
Comment (rs028) [reply doc/latex/Development.tex:61]: As far as I understand it is still not working properly. James was looking into it, but I don't have an update. I will ask.

**Comment by spco** [comment]
I'll have a play around to see what we can do without it indeed becoming a maintenance nightmare!

File: model/output/.gitignore [hunk 3]
```diff
@@ -1,6 +1,2 @@
-# Ignore everything in this directory
-*
-
-# Except this file and reactionRates directory
-!.gitignore
-!reactionRates
+# Ignore output files in this directory
+*.output
```
[the remark this answers is not in the document]
Comment (rs028) [reply model/output/.gitignore:2]: I don't think we were tracking those files. There is a `.gitignore` in the `reactionRates/` directory.

**Comment by spco** [comment]
Ok, I'm going to revert the last few commits and abandon gfortran-8 on either platform for now. The reasons:

(1) on ubuntu, gfortran-8 is no longer in the package managers (minimum supported is 9). The only alternative is to compile from source, which is possible but will take a few minutes of compute time, and I don't have time to get that working right now.

(2) on macOS, for some reason the CVODE install insists that gfortran-8 is not a functioning fortran compiler. It then deactivates the `FCMIX` flag, which means that the `*_fcvode.a` libraries are not built. It then fails later at the stage of running the `atchem2` executable as it can't link to the nonexistent libraries. I'm not sure what would be required to make CVODE think gfortran-8 _is_ functional, so I'll drop that for now too due to time.

So we should end up with ubuntu-22.04, macos-11 and macos-12, all running gfortran-9, 10 and 11.

It would still be desirable to get gfortran-8 running on at least one architecture in future, but not crucial.

**Comment by rs028** [comment]
Sounds reasonable. And thanks for sorting this out. I will now try to push my last changes and (if you are done) merge this branch.

**Pull request merged**